wazimap_ng/datasets/tasks/indicator_data_extraction.py

```diff
@@ -9,8 +9,15 @@
 
 
 @transaction.atomic
-def indicator_data_extraction(indicator, *args, universe={}, **kwargs):
+def indicator_data_extraction(indicator, *args, **kwargs):
     indicator.indicatordata_set.all().delete()
+
+    universe = {}
+    if indicator.universe and isinstance(indicator.universe.filters, dict):
+        universe = {
+            f"data__{key}": value for key, value in indicator.universe.filters.items()
+        }
+
     geography_ids = (
         models.DatasetData.objects.filter(dataset=indicator.dataset)
         .filter(**universe)
```

Comment:
It looks like this is re-implementing what we have in wazimap_ng/datasets/models/datasetdata.py

```python
class DatasetDataQuerySet(models.QuerySet):
    def filter_by_universe(self, universe):
        filters = universe.filters
        if filters and isinstance(filters, dict):
            filters = {f"data__{k}": v for k, v in filters.items()}
            return self.filter(**filters)
        else:
            return self
```

That stopped getting called in 

```
commit 496010d1bb0aa9edd6d6f8ae154dc216a220ed1b
Author: Adi Eyal <adi@openup.org.za>
Date:   Tue Mar 16 15:07:18 2021 +0200

    Refactored indicator_data_extract
    
    Instead of calculating groups, this function simply creates new IndicatorData objects with raw data.
    
    This commit breaks old tests which will be fixed in the following commit.
```

As a matter of style, if we don't use the existing code above, how about renaming `universe` to `filters` or `datasetdata_filters`? Because it's not a Universe instance, it's a dict of filter keys and values.

Reply:
so it looks like generating and passing the universe just got lost in the refactoring, and there wasn't a test to detect that.

I'm happy with your implementation if we just change the name as suggested, but also delete the old implementation.